google-cloud-spanner/src/test/java/com/google/cloud/spanner/OptionsTest.java

```diff
@@ -510,4 +517,68 @@ public void testTransactionOptionsPriorityEquality() {
     assertNotEquals(options2, options3);
     assertNotEquals(options2, options4);
   }
+
+  @Test
+  public void updateOptTest() {
+    String tag = "app=spanner,env=test";
+    Options opts = Options.fromUpdateOptions(Options.tag(tag));
+
+    assertThat(opts.toString()).isEqualTo("tag: " + tag + " ");
+    assertThat(opts.hasTag()).isTrue();
+    assertThat(opts.tag()).isEqualTo(tag);
+    assertThat(opts.hashCode()).isEqualTo(-2118248262);
+  }
+
+  @Test
+  public void updateEquality() {
+    Options o1;
+    Options o2;
+    Options o3;
+
+    o1 = Options.fromUpdateOptions();
+    o2 = Options.fromUpdateOptions();
+    assertThat(o1.equals(o2)).isTrue();
+
+    o2 = Options.fromUpdateOptions(Options.tag("app=spanner,env=test"));
+    assertThat(o1.equals(o2)).isFalse();
+    assertThat(o2.equals(o1)).isFalse();
+
+    o3 = Options.fromUpdateOptions(Options.tag("app=spanner,env=test"));
+    assertThat(o2.equals(o3)).isTrue();
+
+    o3 = Options.fromUpdateOptions(Options.tag("app=spanner,env=stage"));
+    assertThat(o2.equals(o3)).isFalse();
+  }
+
+  @Test
+  public void transactionOptTest() {
+    String tag = "app=spanner,env=test";
+    Options opts = Options.fromTransactionOptions(Options.tag(tag));
```

Comment:
nit: please don't abbreviate `Options` to `opts`. (I know this was already like this, but if you are to make any other changes, could you please fix this as well).

Reply:
Done.